docs/reference/ingest/ingest.asciidoc

```diff
@@ -1,23 +1,22 @@
 [[ingest]]
-== Ingest Plugin
+== Ingest
 
-The ingest plugin can be used to pre-process documents before the actual indexing takes place.
-This pre-processing happens by the ingest plugin that intercepts bulk and index requests, applies the
+Ingest can be used to pre-process documents before the actual indexing takes place.
+This pre-processing happens by Ingest that intercepts bulk and index requests, applies the
 transformations and then passes the documents back to the index or bulk APIs.
 
-The ingest plugin is disabled by default. In order to enable the ingest plugin the following
+Ingest is enabled by default. In order to disable ingest the following
```

Comment:
we should rather talk about this as a new node type more than saying ingest enabled like we did when it was a plugin?